Pipfile

```diff
@@ -30,7 +30,7 @@ mergedeep = "*"
 mypy = "==0.782"
 networkx = "*"
 nltk = "*"
-pandas = "*"
+pandas = "<1.3.4"
```

Comment:
why?

Reply:
Because pandas new version are depended some lib that required python 3.10 at minimum
i discussed it with @jochman and this should be the solution
may be should be in separate PR?

Reply:
it should be in a separate pr